src/renderers/SymbolInstanceRenderer.js

```diff
@@ -71,7 +71,7 @@ const extractOverridesReducer = (
   if (layer._class === 'symbolInstance') {
     return overrides.concat({
       ...overrideProps(layer),
-      symbolId: layer.symbolID,
```

Comment:
is that a breaking change?

Reply:
So this is actually misleading due to some inconvenient naming conventions which I plan to address... These are actually the `overridables` and not the overrides. The `overridables` are only used internally so it shouldn't be a problem.

Elsewhere in this PR however, I removed the `masterName` static prop on the created `SymbolInstance`. This would probably be considered a breaking change. There is no reason I can't put it back in.

I have however hit a few restraints with the current symbols implementation that will REQUIRE breaking changes.

Reply:
`masterName` is not a prop right? just a class field I believe. Let's put it back and I'll merge the PR

Reply:
It was a prop on the `symbolinstance` but that is only used internally so I believe it is safe to remove there.

It was additionally set as a static property on the `React.Component` returned in the factory used to generate user facing `SymbolInstance`. I have just added it back here as there is a chance someone was relying upon it being there.